include/mbedtls/mbedtls_config.h

```diff
@@ -3073,6 +3083,87 @@
 
 /** \} name SECTION: mbed TLS modules */
 
+/**
+ * \name SECTION: General configuration options
+ *
+ * This section allows for the setting of module specific sizes and
```

Comment:
It's not really obvious what "module-specific-sizes" you are talking about here, as there's no size defined in this section.

Reply:
That was copypasta from an existing section. Fixed.